### PR TITLE
Toaster - Adding pause message's timer on hover feature

### PR DESCRIPTION
**Pull request by EAlexRojas**

### CODE REVIEW

File: ToasterMessage.js [hunk 1]
```diff
@@ -221,6 +221,44 @@ define(["dcl/dcl",
...
+		}
+
+		function _pointerLeaveHandler() {
+			if (hovering) {
+				hovering = false;
+				element._timer.resume();
+			}
+		}
+
+		this.isEnabled = false;
```
Comment (cjolif) [inline ToasterMessage.js]: Is that flag meant to be public? If not it should start with underscore.

File: ToasterMessage.js [hunk 2]
```diff
@@ -221,6 +221,44 @@ define(["dcl/dcl",
 		}.bind(this));
 	};
 
+	var PauseTimerOnHover = function (element) {
+
+		var hovering = false;
+		function _pointerOverHandler() {
+			console.log("hover event");
```
Comment (cjolif) [inline ToasterMessage.js]: don't leave traces in final code.

Comment (EAlexRojas) [inline ToasterMessage.js]: Sorry, I'll fix that

File: ToasterMessage.js (hunk 1 above)
Comment (EAlexRojas) [inline ToasterMessage.js]: Yes, as the feature can be enabled/disabled, I think it could be useful to the user to know that value.

**Comment by tkrugg** [comment]
merged in https://github.com/ibm-js/deliteful/commit/286b89180bfa9e59efc32a63f29baeb640513f1f

**Pull request closed without merging**